proposals/xxxx-experimental-dxil.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "XXXX - Experimental DXIL"
+params:
+  authors:
+    - V-FEXrt: Ashley Coleman
+    - llvm-beanz: Chris Bieneman
+    - tex3d: Tex Riddell
+  sponsors:
+    - V-FEXrt: Ashley Coleman
+  status: Under Consideration
+---
+
+* Planned Version: 2021
```

Comment:
2021 here sounds like a language version, which is independent of DXIL versioning.

I think you mean that this is planned for SM 6.10?